samples/apps/spring-petclinic/.patch/pom.xml.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/pom.xml b/pom.xml
+index ae02829..ef52b15 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -30,6 +30,8 @@
+     <nohttp-checkstyle.version>0.0.4.RELEASE</nohttp-checkstyle.version>
+     <spring-format.version>0.0.27</spring-format.version>
+ 
++    <spring-data-jdbc.version>1.2.1.RELEASE</spring-data-jdbc.version>
++
+   </properties>
+ 
+   <dependencies>
+@@ -46,6 +48,10 @@
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-data-jpa</artifactId>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.boot</groupId>
++      <artifactId>spring-boot-starter-jdbc</artifactId>
++    </dependency>
+     <dependency>
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-web</artifactId>
+@@ -58,6 +64,11 @@
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-thymeleaf</artifactId>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.cloud</groupId>
++      <artifactId>spring-cloud-bindings</artifactId>
++      <version>1.8.0</version>
++    </dependency>
+     <dependency>
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-test</artifactId>
+@@ -86,6 +97,17 @@
+       <artifactId>postgresql</artifactId>
+       <scope>runtime</scope>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.data</groupId>
++      <artifactId>spring-data-jdbc-core</artifactId>
++      <version>${spring-data-jdbc.version}</version>
++      <exclusions>
++        <exclusion>
++          <groupId>org.springframework</groupId>
++          <artifactId>*</artifactId>
++        </exclusion>
++      </exclusions>
++    </dependency>
+ 
+     <!-- caching -->
+     <dependency>
+@@ -253,6 +275,14 @@
+         <enabled>true</enabled>
+       </snapshots>
+     </repository>
++    <repository>
++      <snapshots>
++        <enabled>false</enabled>
++      </snapshots>
++      <id>spring-libs-private-release</id>
++      <name>libs-private-release</name>
++      <url>https://repo.spring.io/artifactory/libs-private-release</url>
++    </repository>
```

Comment:
why do we need this?

Reply:
This maven repo is where `spring-cloud-binding` library is available

Reply:
hm, it sounds like some private repo, not official one.. how about that we use `https://repo.spring.io/releases` instead?

Reply:
Good catch! I missed that, I assumed the release repo was already there so i looked for it elsewhere. But it wasn't. I'll use the release repo instead. thx